openfisca_aotearoa/parameters/general/age_of_majority.yaml

```diff
@@ -1,7 +1,8 @@
 description: Age of majority (in years)
+reference: http://www.legislation.govt.nz/act/public/1970/0137/latest/whole.html
 values:
   # This parameter has last changed on the 1st of Jan 1971.
-  1945-01-01:
-    value: 21.0
-  1971-01-01:
-    value: 18.0
+  1970-12-02:
+    value: 20.0
+  # proposed 2018 https://www.parliament.nz/en/pb/bills-and-laws/proposed-members-bills/document/52HOH_MEMBILL007_1/age-of-majority-attainment-at-18-years-amendment-bill
+  # value: 18.0
```

Comment:
> Very interesting! Just FYI, this could be in the future implemented as a [reform](http://openfisca.org/doc/coding-the-legislation/reforms.html) to assess the impact of this change on the population 😃

Reply:
> I also was interested if OpenFisca has a particular way to flag future possible changes